File: mc/rmcp/src/main/java/calebxzhou/rdi/mc/common2/mcp/RMcpMenuDropData.java
package calebxzhou.rdi.mc.common2.mcp;

public record RMcpMenuDropData(
        String format,
        boolean dryRun,
        int slot,
        int requestedCount,
        int droppedCount,
        boolean changed,
        RMcpMenuData.Slot beforeSlot,
        RMcpMenuData.Slot afterSlot,
        RMcpMenuData menu
) {
}

File: mc/rmcp/common/src/main/kotlin/calebxzhou/rdi/mc/common2/mcp/RMcpMenuDropData.kt
package calebxzhou.rdi.mc.common2.mcp

@JvmRecord
data class RMcpMenuDropData(
    val dryRun: Boolean,
    val slot: Int,
    val requestedCount: Int,
    val droppedCount: Int,
    val changed: Boolean,
    val beforeSlot: RMcpMenuData.Slot,
    val afterSlot: RMcpMenuData.Slot,
    val menu: RMcpMenuData?
)
